src/node_internals.h

```diff
@@ -264,6 +264,10 @@ class ThreadPoolWork {
   virtual void DoThreadPoolWork() = 0;
   virtual void AfterThreadPoolWork(int status) = 0;
 
+  Environment* get_env() const {
```

Comment:
```suggestion
  Environment* env() const {
```
The house style for Node.js core is `foo()` and `set_foo()` for the getter and setter respectively.

Reply:
@bnoordhuis Thanks for letting me know, changing this generates errors in node_crypto.cc with overloaded env() functions though

Reply:
I'm only guessing because I haven't tried it yet, but we can likely resolve the errors in node_crypto.cc using explicitly scoped calls to `env()`... like `ThreadPoolWork::env()` rather than just `env()`, but I'll take a look at it tomorrow to see if that would work

Reply:
@jasnell That would add a lot of overhead… We had a similar situation once in the past: There can be no `StreamBase::env()` method because that would also conflict with `BaseObject::env()`, so we named it `StreamBase::stream_env()`. I’m okay with following that pattern, but I’m also okay with the PR in its current state

Reply:
+1 I'm good with it also!

Reply:
Well, I'm not. :-) It's working around something that's fairly easy to fix.

I started typing up an explanation but then I realized it's probably easier to make the change than trying to convey it. Can you take a look at #31588?